etc/avocado/conf.d/vt_joblock.conf

```diff
@@ -0,0 +1,4 @@
+[plugins.vtjoblock]
```

Comment:
Right!
On May 30, 2016 12:54 PM, "Lukáš Doktor" <notifications@github.com> wrote:

> In etc/avocado/conf.d/vt_joblock.conf
> <https://github.com/avocado-framework/avocado-vt/pull/543#discussion_r65088056>
> :
>
> > @@ -0,0 +1,4 @@
> > +[plugins.vtjoblock]
>
> Also as mentioned in d8b4379
> <https://github.com/avocado-framework/avocado-vt/pull/484/commits/d8b43798ffe8b568878efcc8452180d23b72897d>
> would you please add this file to make link target?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/avocado-framework/avocado-vt/pull/543/files/223f5509233f77c0a8af0e13f5a5175c37e4b4f3#r65088056>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAbSypqhicTuHD3wuNe_zJ3f1dWmxH8sks5qGwgjgaJpZM4IotF7>
> .
>